astropy/io/registry.py

```diff
@@ -318,12 +320,20 @@ def register_identifier(data_format, data_class, identifier, force=False):
         _identifiers[(data_format, data_class)] = identifier
     else:
         raise IORegistryError("Identifier for format '{0}' and class '{1}' is "
-                              'already defined'.format(data_format,
-                                                       data_class.__name__))
+                              'already defined.'.format(data_format,
+                                                        data_class.__name__))
 
 
 def identify_format(origin, data_class_required, path, fileobj, args, kwargs):
-    # Loop through identifiers to see which formats match
+    """Loop through identifiers to see which formats match.
+
+    For the parameter description see `register_identifier`.
```

Comment:
@taldcroft I don't understand? I thought the list in http://docs.astropy.org/en/stable/api/astropy.io.registry.register_identifier.html#astropy.io.registry.register_identifier is exactly the parameter description, at least `origin, path, fileobj, args, kwargs`.

I now realize that `data_class_required` is not listed there. But I don't think you meant the one parameter.